ngraph_creator/operations/src/Pad_V2.cpp

```diff
@@ -65,18 +65,18 @@ std::shared_ptr<ngraph::Node> Pad_V2::createNode() {
         pad_value = DequantizeNode(pad_value, inputIndex, ngraph::element::f32);
     }
 
-    const auto axisNode = createConstNode(ngraph::element::i32, {}, convertToVector(1));
-    auto paddingsSplitNode =
-        std::make_shared<ngraph::opset3::Split>(paddings, axisNode, 2)->outputs();
-
-    const auto shapeNode = createConstNode(
-        ngraph::element::i32, {1}, convertToVector((int32_t)getInputOperandDimensions(0).size()));
-
-    std::shared_ptr<ngraph::Node> pads_begin =
-        std::make_shared<ngraph::opset3::Reshape>(paddingsSplitNode[0], shapeNode, true);
-
-    std::shared_ptr<ngraph::Node> pads_end =
-        std::make_shared<ngraph::opset3::Reshape>(paddingsSplitNode[1], shapeNode, true);
+    const auto& paddingsOperandIndex = sModelInfo->getOperationInput(mNnapiOperationIndex, 1);
+    // Fetch the 2D paddings as a 1D vector, and then split it into 2
+    auto paddings_2d = sModelInfo->GetConstVecOperand<int32_t>(paddingsOperandIndex);
+    auto half_size = paddings_2d.size() / 2;
+    std::vector<int32_t> paddings_0(half_size);
+    std::vector<int32_t> paddings_1(half_size);
+    for (int i = 0; i < half_size; i++) {
```

Comment:
@akodanka Can u help me understand why we can not use split along the last dimension and get two arrays?

Reply:
The result of the split operation would be available only at runtime. Hence, this would cause dynamic output dimensions in the graph.

Reply:
@akodanka  Thanks for the explanation. So nGraph API requires the padding dimensions at the time of graph creation.